cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -62,6 +62,34 @@
   </test>
 
   <!-- aux_clm test suite failures -->
+  <test name="LII2FINIDATAREAS_D_P256x2_Ld1.f09_t232.I1850Clm60BgcCrop.derecho_intel.clm-default">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>#3252</issue>
+      <comment>Works with finidat = 'ctsm53041_54surfdata_snowTherm_100_pSASU.clm2.r.0161-01-01-00000.nc' and fails with finidat = 'ctsm53041_54surfdata_snowTherm_100_pSASU.clm2.r.0161-01-01-00000_64bitoffset.nc'.</comment>
+    </phase>
+  </test>
+  <test name="LII2FINIDATAREAS_D_P256x2_Ld1.f09_t232.I1850Clm60BgcCrop.derecho_intel.clm-default--clm-matrixcnOn_ignore_warnings">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>#3252</issue>
+      <comment>Works with finidat = 'ctsm53041_54surfdata_snowTherm_100_pSASU.clm2.r.0161-01-01-00000.nc' and fails with finidat = 'ctsm53041_54surfdata_snowTherm_100_pSASU.clm2.r.0161-01-01-00000_64bitoffset.nc'.</comment>
+    </phase>
+  </test>
+  <test name="SMS_D_Ld5.f09_g17.ISSP126Clm50BgcCrop.derecho_intel.clm-default">
```

Comment:
These should be able to go away with going back to the ctsm5.3 surface datasets.

Reply:
Ok, removing ISSP tests from expected failures.